src/lib/components/common/ISO8601DateTimePicker.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import DateFnsUtils from '@date-io/date-fns';
+import { format } from 'date-fns';
+import moment from 'moment-timezone';
+import { 
+    FormControl
+} from '@material-ui/core';
+import { 
+    DateTimePicker,
+    MuiPickersUtilsProvider 
+} from '@material-ui/pickers';
+
+/*
+    Displays a text field that opens a date-time picker when selected. The date
+    and time are entered and displayed in the local time using the 24 hour
+    clock, however, the activeDrsObject is updated with the date and time
+    converted to UTC time. Seconds cannot be selected, and therefore they are
+    always automatically set to zero. The date and time are displayed in the
+    format 'yyyy-MM-dd HH:mm:ss OOOO' which includes the timezone (offset from
+    GMT). The date-time picker includes a "Today" button which sets the value
+    to the current minute when clicked.
+*/
+const ISO8601DateTimePicker = props => {
+
+    useEffect(() => {
+        moment.tz.setDefault('Asia/Kolkata')
+
+    }, [])
+
+    return (
+        <MuiPickersUtilsProvider utils={DateFnsUtils}>
+            <FormControl fullWidth>
+                <DateTimePicker
+                    margin="normal"
+                    format="yyyy-MM-dd HH:mm:ss OOOO"
+                    id={props.id}
+                    label={props.label}
+                    name={props.name}
+                    value={props.value} 
+                    readOnly={props.readOnly}
+                    showTodayButton
+                    ampm={false}
+                    helperText={props.helperText}
+                    onChange={date => {
+                        date.setSeconds(0, 0);
+                        let utcDate = format(date, "yyyy-MM-dd'T'HH:mm:ss'Z'");
+                        props.setFunction(utcDate);
```

Comment:
Currently this component does not seem to convert between UTC and local time correctly. On the NEW form, the time that is displayed is offset by the difference from the local time to UTC time. For example, on loading the NEW page, the Created Time and Updated Time fields should display the current local time and indicate the timezone (i.e. the difference from GMT), however, they are actually displaying the current local time minus 4 hours (the difference between EST and GMT) and indicate that the timezone is GMT-4:00 (EST). The time submitted to the API matches the current local time but is represented as UTC without being converted.